package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "onespan-sign-node",
+  "version": "0.1.0",
+  "description": "Node.js library for the OneSpan Sign API",
+  "main": "dist/index.js",
+  "repository": "https://github.com/getbreathelife/onespan-sign-node.git",
+  "author": "Breathe Life",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "build": "tsc",
+    "lint": "eslint --quiet ./",
+    "test": "jest -i",
+    "prepublishOnly": "npm run compile && npm run test"
+  },
+  "keywords": [
+    "onespan",
+    "sign",
+    "node",
+    "typescript",
+    "javascript"
+  ],
+  "dependencies": {
+    "node-fetch": "^3.1.0"
```

Comment:
Using `node-fetch` instead of `axios` since this package is intended to run on the Node.js environment and doesn't require making `XMLHttpRequests` like on the browser. `node-fetch` uses Node.js [HTTP API](https://nodejs.org/api/http.html) internally